bucket/wechat.json

```
@@ -1,61 +1,52 @@
 {
-    "version": "3.9.12.45",
-    "description": "Free messaging and calling app by Tencent",
-    "homepage": "https://pc.weixin.qq.com/",
+    "version": "4.0.3.36",
+    "description": "Wechat 4.0, Free messaging and calling app by Tencent, Unified cross-device experience",
+    "homepage": "https://windows.weixin.qq.com/",
     "license": {
         "identifier": "Proprietary",
-        "url": "https://www.wechat.com/en/service_terms.html"
+        "url": "https://weixin.qq.com/cgi-bin/readtemplate?lang=zh_CN&t=weixin_agreement"
```

```suggestion
        "url": "https://www.wechat.com/en/service_terms.html"
```

It will redirect to `https://www.wechat.com/en/service_terms.html` finally.